src/scss/components/_subtitle.scss

```diff
@@ -1,4 +1,4 @@
-.sparkeats-subtitle {
+.subtitle {
```

Comment:
The name `Subtitle` has been bothering me. It feels to general, like it needs to be qualified. Would `SiteSubtitle` be a better name for the component and related styles and would align with [`SiteHeader`](https://github.com/sparkbox/sparkeats/blob/main/src/components/SiteHeader.tsx). What do you think?

Reply:
Updated in d97c677e9a8d528d0e7ba6ef17b3578feb1568c5.